go/install/install_osx.go

```diff
@@ -557,6 +566,75 @@ func InstallKBFS(context Context, binPath string, force bool, timeout time.Durat
 	return nil
 }
 
+// Return where the NativeMessaging host manifest lives on this platform.
+func kbnmManifestPath() string {
+	// Find path to write JSON into, one of:
+	// "/Library/Google/Chrome/NativeMessagingHosts"
+	// "$HOME/Library/Application Support/Google/Chrome/NativeMessagingHosts"
+	// XXX: Are we always root?
+	return "/Library/Google/Chrome/NativeMessagingHosts"
```

Comment:
Please use the "$HOME/Library/Application Support/Google/Chrome/NativeMessagingHosts" dir?

Also NativeMessagingHosts dir doesn't exist for me there, so write would probably fail on my machine